WooCommerce/Classes/Authentication/Jetpack Setup/Native Jetpack Setup/LoginJetpackSetupViewModel.swift

```diff
@@ -201,10 +211,11 @@ private extension LoginJetpackSetupViewModel {
             guard let self else { return }
             switch result {
             case .success(let url):
+                self.analytics.track(.loginJetpackSetupFetchJetpackConnectionURLSuccessful)
                 self.jetpackConnectionURL = url
                 self.shouldPresentWebView = true
             case .failure(let error):
-                // TODO: add tracks
+                self.analytics.track(.loginJetpackSetupFetchJetpackConnectionURLFailed)
```

Comment:
Similar as above:
```suggestion
                self.analytics.track(.loginJetpackSetupFetchJetpackConnectionURLFailed, withError: error)
```

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/8187/commits/30a867f9bac22e88b0c4b1b7b31d914a4194abb6